webextension/background.js

```diff
@@ -82,10 +82,10 @@ function send_css(tab_id, iframe_css) {
     browser.tabs.sendMessage(tab_id,
 			     { generated_css: generated_css,
 			       user_css: prefs.useCustomCSS ?
-			                 prefs.customCSS : "" });
+			                 prefs.customCSS : "" }).catch(error => console.log("error with tab", tab_id, error));
 }
 function broadcast_css(iframe_css) {
-    browser.tabs.query({}, function(tabs) {
+    browser.tabs.query({url: "<all_urls>"}, function(tabs) {
```

Comment:
the property `url: "<all_urls>` allows to bypass tabs with `about:..` url and avoids unneeded work and error at startup